### PR TITLE
Adding sql bindings for pg in profiler results

### PR DESCRIPTION
**Pull request by Neznauy**
Hi,
according to https://github.com/MiniProfiler/rack-mini-profiler/issues/288 , I added frontend to show it. Also, there was no transfer bindings, and bindings had only value and no name, in pg at least.

### CODE REVIEW

**Comment by SamSaffron** [comment]
can I see some screenshots of the change?

**Comment by Neznauy** [comment]
max_sql_param_length = 1
eq1
max_sql_param_length = 0, the same as before my changings
eq0
max_sql_param_length = 5
eq5
max_sql_param_length = nil
eqnil

**Comment by SamSaffron** [comment]
looks good! we should probably default param length to 20 or so.

File: lib/patches/db/pg.rb [hunk 1]
```diff
@@ -92,11 +92,29 @@ def async_exec(*args,&blk)
...
-    record       = ::Rack::MiniProfiler.record_sql(args[0], elapsed_time)
+    record       = ::Rack::MiniProfiler.record_sql(args[0], elapsed_time, get_binds(*args))
     result.instance_variable_set("@miniprofiler_sql_id", record) if result
 
     result
   end
-
+  
+  def get_binds(*args)
+    return if args[1].nil?
```
Comment (SamSaffron) [inline lib/patches/db/pg.rb]: hmmm this concerns me performance wise, we are building hh without shortening it at all. 

this patch should only grab the text the ui needs

Comment (SamSaffron) [inline lib/patches/db/pg.rb]: also lots of regexes here which should be constants, I would also do a while loop here cause perf is super sensitive for profilers

**Comment by SamSaffron** [comment]
Sorry, I am reverting this for now and doing another release, it is somewhat buggy and completely untested, totally happy to add this if we have a proper set of tests for it.

**Pull request merged**